Modules/core/Model/CoreInstall.php

```diff
@@ -477,6 +477,10 @@ public function upgrade_v2_v3() {
         $this->runRequest($sql);
         $sql = "update se_project_service set `date`=null where `date`='0000-00-00'";
         $this->runRequest($sql);
+        $sql = "alter table se_project modify column date_open date NULL";
```

Comment:
logiquement il faut mettre cela plus haut car on met les dates à null

        $sql = "update se_project set date_open=null where date_open='0000-00-00'";
        $this->runRequest($sql);
        $sql = "update se_project set date_close=null where date_close='0000-00-00'";
        $this->runRequest($sql);
        $sql = "update se_project_service set `date`=null where `date`='0000-00-00'";